functorch/_src/aot_autograd.py

```diff
@@ -783,50 +726,18 @@ def flat_fn(*flat_tensor_args):
 
             compiled_fn = create_aot_dispatcher_function(
                 flat_fn,
-                flat_tensor_args,
+                flat_args,
                 aot_config,
             )
             cached_res = (compiled_fn, out_spec)
 
-            # Save the compiled_fn in the cache
-            compile_cache.insert(
-                fn_id,
-                fw_compiler_id,
-                bw_compiler_id,
-                num_tensor_args,
-                hasher_type,
-                cached_res,
-                *flat_args_for_cache,
-            )
-
         cached_fn, out_spec = cached_res
```

Comment:
rename to compiled_fn and delete cached_res above?

Reply:
We still cache the saved function, we just never recompile now.